script/sync

```diff
@@ -134,6 +134,30 @@ def sync_ruby(version)
   puts "done."
 end
 
+def sync_node(version)
+  s3_key = "nodes/Darwin/#{s3_cellar}#{os}/#{version}.tar.bz2"
```

Comment:
Can you check files with `otool -L` to see if there's any linkage to the `/opt/boxen/homebrew` or `/usr/local` directories? Thanks!